focuspoints.lrdevplugin/DefaultDelegates.lua

```diff
@@ -109,6 +109,10 @@ function DefaultDelegates.getShotOrientation(photo, metaData)
   local orgPhotoW, orgPhotoH = parseDimens(dimens) -- original dimension before any cropping
 
   local metaOrientation = ExifUtils.findFirstMatchingValue(metaData, { "Orientation" })
+  if metaOrientation == nil then
```

Comment:
i merged, but wouldn't the string match just fail all conditions and return 0 at the end?